clans/src/main/java/me/mykindos/betterpvp/clans/progression/perks/FieldExperienceMiningPerk.java

```diff
@@ -0,0 +1,89 @@
+package me.mykindos.betterpvp.clans.progression.perks;
+
+import com.google.inject.Inject;
+import com.google.inject.Singleton;
+import lombok.extern.slf4j.Slf4j;
+import me.mykindos.betterpvp.clans.clans.ClanManager;
+import me.mykindos.betterpvp.core.config.Config;
+import me.mykindos.betterpvp.core.listener.BPvPListener;
+import me.mykindos.betterpvp.progression.Progression;
+import me.mykindos.betterpvp.progression.model.ProgressionPerk;
+import me.mykindos.betterpvp.progression.model.ProgressionTree;
+import me.mykindos.betterpvp.progression.model.stats.ProgressionData;
+import me.mykindos.betterpvp.progression.tree.mining.Mining;
+import me.mykindos.betterpvp.progression.tree.mining.event.ProgressionMiningEvent;
+import org.bukkit.entity.Player;
+import org.bukkit.event.EventHandler;
+import org.bukkit.event.Listener;
+
+import java.util.function.LongUnaryOperator;
+
+@Slf4j
+@Singleton
+public class FieldExperienceMiningPerk implements Listener, ProgressionPerk {
```

Comment:
We already have a global fields experience multiplier, though it doesn't happen to be a perk. I'm not sure if I want to separate it into a perk, however. I kind of want to incentivize everyone to mine at fields to keep it similar to Fishing on how the main activity has to be done in fields.

Reply:
This makes sense, this is designed to function with other experience modifiers. My next project is to add enable/disable and a standardized config for mining. I think it makes more sense as a perk.

Reply:
Removed